### PR TITLE
fixes PHP version parse even when it has warnings

### PR DESCRIPTION
**Pull request by herbertpimentel**
address to issue discribed in https://github.com/felixfbecker/php-language-server/issues/237

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
     }
 
     // Parse version and discard OS info like 7.0.8--0ubuntu0.16.04.2
-    const match = stdout.match(/^PHP ([^\s]+)/);
+    const match = stdout.match(/PHP ([^\s]+[\d]+)/);
```
Comment (felixfbecker) [inline src/extension.ts]: I get why you are removing the caret, but why are you adding a constraint for a sequence of numbers in the end, with a character class that contains only a digit?

Comment (herbertpimentel) [inline src/extension.ts]: to make sure I wont get any other match when parsing the ouput version, checkout this regex test https://regex101.com/r/R0NUN7/1 if we remove the extra \d I afraid we may run into some unexpected match

Comment (felixfbecker) [inline src/extension.ts]: FYI `[\d]` is equivalent to `\d`
I think we should keep the regex like it was before, but just add the multiline flag, it should fix your issue.

Comment (herbertpimentel) [inline src/extension.ts]: sure, that is awesome, I agree that is a good idea only add multiline flag

**Pull request merged**